lib/Service/ReportService.php

```diff
@@ -124,6 +145,12 @@ public function listReports()
 
         $this->log->debug('Reports found in total: '.count($reports));
 
+        // Get reports that are allowed by connectors
+        $event = new ConnectorReportEvent();
```

Comment:
@dasgarner I am getting an error in  test ReportScheduleDataTest because of the dispatched event, how do we resolve this in the test https://github.com/ifarzana/xibo-cms/blob/campaignPop/tests/integration/ReportScheduleDataTest.php#L146?